success-criteria.md

```diff
@@ -337,21 +309,23 @@ stakeholder group.
 
         -   Free elections protected against foreign manipulation
 
-            -   While free speech requires allowing speech the majority does not
```

Comment:
I am unclear what this has to do with the interests of society when expressed at this level of detail or why it is placed here. This is a minor point but is part of a much more major concern: free speech requires clear sourcing and identity at a publisher level. I have rephrased this point and put under Freedom of the press as a major point with the rephrase as a secondary point.